src/battle_script_commands.c

```diff
@@ -11094,6 +11095,7 @@ static void Cmd_pickup(void)
                     SetMonData(&gPlayerParty[i], MON_DATA_HELD_ITEM, ITEM_HONEY);
```

Comment:
I think this way you're trying to make a pointer from an integer without a cast. Passing the 3rd arg that way.